src/fundus/publishers/base_objects.py

```diff
@@ -193,3 +206,15 @@ def __len__(cls) -> int:
             int: The number of publishers.
         """
         return len(list(iter(cls)))
+
+    def __str__(self) -> str:
+        enum_mapping = self.get_publisher_enum_mapping()
+        enum_mapping_keys = enum_mapping.keys()
+        representation = f"The entire Publisher Collection consists of {len(self)} publishers from {len(enum_mapping_keys)} countries:"
```

Comment:
```suggestion
        representation = f"The {self.__name__!r} consists of {len(self)} publishers from {len(enum_mapping_keys)} countries:"
```

Or `collection` instead.